plugins/CoreHome/stylesheets/layout.less

```
@@ -461,6 +461,34 @@ nav {
   }
 }
 
+.badge-menu-item-container {
+  line-height: inherit;
+  position: relative;
+}
+.badge-menu-item {
+  position: absolute;
+  top: 12px;
+  left: 23px;
+  line-height: 1.25;
+  background-color: rgba(212, 19, 13, 1);
+  color: #fff;
+  border-radius: 8px;
+  padding: 4px 6px;
+  font-size: 9px;
+  font-weight: bold;
+}
+
+.whatisnew-changelist {
+  overflow: scroll;
```

Showing the horizontal scroll bar imho doesn't make much sense here.
```suggestion
  overflow-y: scroll;
```